src/ngx_http_lua_worker.c

```diff
@@ -107,6 +107,7 @@ ngx_http_lua_ffi_enable_privileged_agent(char **err)
                                            ngx_core_module);
 
     ccf->privileged_agent = 1;
+    ccf->privileged_agent_connections = connections;
```

Comment:
This line can't even be built.

Reply:
It depends on the sister PR: https://github.com/openresty/openresty/pull/686